tests/test_pipelines/test_projects.py

```diff
@@ -3,7 +3,6 @@
 from unittest.mock import patch
 from tests.test_analysisdriver import TestAnalysisDriver, NamedMock
 from analysis_driver.pipelines import projects
-from analysis_driver.exceptions import PipelineError
 
 test_projects = os.path.join(TestAnalysisDriver.assets_path, 'test_projects')
 relatedness_outfiles = os.path.join(test_projects, 'relatedness_outfiles')
```

Comment:
An extra test here for build_pipeline would be good. It should mock many samples and test that the combining in batches is setup as expected.

Reply:
Implemented